demo-simple/src/main/java/com/novoda/downloadmanager/demo/DemoApplication.java

```diff
@@ -24,6 +25,7 @@ private void createLiteDownloadManager() {
 
         liteDownloadManagerCommands = DownloadManagerBuilder
                 .newInstance(this, handler, R.mipmap.ic_launcher_round)
+                .withAllowedConnectionType(ConnectionType.UNMETERED)
```

Comment:
Can we remove this, I had a problem when I first worked on the project where we had a bunch of these defined in `Application` and they were changing the `download-manager` without me knowing 😄  would prefer to keep just the vanilla in the sample app.

Reply:
https://github.com/novoda/download-manager/pull/318/commits/b1559fd34d40862bf8f63d42f9ab6230c22adcf0